scss/bootstrap/components/_date-picker.scss

```diff
@@ -0,0 +1,41 @@
+.mobile {
+  date-picker {
+    .chr_uib-date-picker {
+      display: none
+    }
+
+    .chr_html5-date-picker {
+      position: relative;
+
+      > input {
+        -webkit-appearance: none;
+        line-height: normal !important;
+
+        &::-webkit-inner-spin-button {
+          -webkit-appearance: none;
+          display: none;
+        }
+      }
+
+      span {
```

Comment:
is this a specific span? it would be good to add a class to it so it's clear what type of element we want to style and what the final result should be.